src/tools/convert/gromacs.py

```diff
@@ -825,6 +825,7 @@ def storeDihedrals(f, types, dihedraltypes, dihedraltypeparams, dihedrals, num_a
     return dihedrals
     
 def storeImpropers(f, types, impropertypes, impropertypeparams, impropers, num_atoms_molecule, num_molecule_copies, molstartindex, atomtype_wildcard):
+    print '#Warning! This parser of the improper angles section of gromacs-format forcefield files is for the Amber forcefield only. Other forcefields may have a different atom ordering.'
```

Comment:
I'll go with a comment pointing the user to the details in gromacs.py, because actually there are a lot of different protein forcefields with various different ways to treat impropers, so if we open bugs for them all we'll have a long list of bugs, without any user ever actually having requested the functionality.